packages/block-editor/src/components/colors-gradients/control.js

```diff
@@ -122,7 +122,7 @@ function ColorGradientControlInner( {
 						{ ... { colors, disableCustomColors } }
 					/>
 				) }
-				{ currentTab === 'gradient' && (
+				{ ( ( currentTab === 'gradient' ) || ( canChooseAGradient && ! canChooseAColor ) ) && (
```

Comment:
I believe the panel is not shown entirely when both are not supported yet, which means we could potentially simplify these tests by removing `canChooseAGradient` right? or do you think it's clearer that way?

Reply:
You are totally right we could simplify things. I applied your suggestion 👍